module/Activity/view/email/activity.phtml

```diff
@@ -0,0 +1,37 @@
+<?php $lang = 'en'; ?>
+<?= $this->translate('Hello,')?><br /><br />
```

Comment:
It indeed would. Can I just make the email in English/Dutch? What has preferences?